lib/puppet/type/netapp_vserver.rb

```diff
@@ -53,6 +53,10 @@
     desc "Vserver comment"
   end
 
+  newparam(:ipspace) do
+    desc "IPspace name. The default value is 'Default'"
```

Comment:
I think if we are not specifying the defaults in the type (but letting the device decide the default) then we shouldn't document it as part of the module.